hw2/01-coins.js

```diff
@@ -1,15 +1,63 @@
-/** Exercise 01 - Coins **/
+/* eslint-disable no-console */
+/** Exercise 01 - Coins * */
 
 const calculateChange = (input) => {
-  // Add your code here
-};
+    let result = `$${input} ==> `
+
+    if (input > 10.0 || input < 0.0) {
```

Comment:
some of the same conditions are checked twice here. It would be a little more brief to do something like:
```javascript
  if (input > 10) {
    return result.concat("Error: the number is too large");
  } else if (input < 0) {
    return result.concat("Error: the number is too small");
  }
```

Reply:
alternatively, if you don't like early returns, this could be split into the three conditions <0, 0-10, >10:

```javascript
const calculateChange = (input) => {
  let result = `$${input} ==> `;

  if (input > 10) {
    result = result.concat("Error: the number is too large");
  } else if (input < 0) {
    result = result.concat("Error: the number is too small");
  } else {
    let total = input * 100;
    ...
    //rest of the work
  }

  return result;
};

```